nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/layers/convolution/DepthwiseConv2D.java

```diff
@@ -55,10 +53,17 @@
 public class DepthwiseConv2D extends DynamicCustomOp {
 
     protected Conv2DConfig config;
+    protected SDVariable in;
+    protected SDVariable weights;
+    protected SDVariable bias;
+
 
     public DepthwiseConv2D(@NonNull SameDiff sameDiff, @NonNull SDVariable input,
                            @NonNull SDVariable weights, SDVariable bias, @NonNull Conv2DConfig conv2DConfig) {
         this(sameDiff, wrapFilterNull(input, weights, bias), conv2DConfig);
+        this.in = input;
+        this.weights = weights;
+        this.bias = bias;
```

Comment:
We shouldn't need the SDVariable in/weight/bias fields.
We can get them again via arg(int) and args() methods.